### PR TITLE
[YUNIKORN-1115] document changes from YUNIKORN-1093

### PR DESCRIPTION
**Pull request by steinsgateted**
From [YUNIKORN-1110](https://issues.apache.org/jira/projects/YUNIKORN/issues/YUNIKORN-1110)
Document changes from [YUNIKORN-1093](https://issues.apache.org/jira/browse/YUNIKORN-1093)

### CODE REVIEW

**Review by chia7712 — COMMENTED** [review]
@steinsgateted thanks for this patch. a couple of comments below.

**Comment by steinsgateted** [comment]
@chia7712 Thank you for your review.
We fix the palcment in YUNIKORN-998.

File: docs/api/scheduler.md [hunk 1]
```diff
@@ -423,6 +423,19 @@ In the example below there are three allocations belonging to two applications.
...
+        "usedResource": "[]",
+        "maxUsedResource": "[]",
+        "partition": "default",
+        "queueName": "",
+        "submissionTime": 1646757410878163500,
+        "finishedTime": 1646757410878225400,
+        "allocations": [],
+        "applicationState": "Rejected",
+        "user": "nobody",
+        "rejectedMessage": "failed to place application application-0003: application rejected: no placment rule matched"
```
[the remark this answers is not in the document]
Comment (wilfred-s) [reply docs/api/scheduler.md:437]: Will fix the text in the doc on commit. The code has been fixed as part of YUNIKORN-998 (confirmed)

**Review by wilfred-s — APPROVED** [review]
LGTM
@chia7712 I'll commit if you are OK with the changes

**Comment by steinsgateted** [comment]
Thank you for the explanation. 
Create [YUNIKORN-1137](https://issues.apache.org/jira/projects/YUNIKORN/issues/YUNIKORN-1137) for placment

**Pull request closed without merging**